src/qt/createassetdialog.cpp

```diff
@@ -67,6 +71,30 @@ void CreateAssetDialog::setUpValues()
     CheckFormState();
 
     ui->unitExampleLabel->setStyleSheet("font-weight: bold");
+
+    // Setup the asset types
+    QStringList list;
+    list.append(tr("Main Asset"));
+    list.append(tr("Sub Asset"));
+//    list.append(tr("Unique Asset"));
+    ui->assetType->addItems(list);
+    type = ISSUE_ROOT;
+    ui->assetTypeLabel->setText(tr("Asset Type") + ":");
+
+    // Setup the asset list
+    ui->assetList->hide();
+    std::vector<std::string> names;
+    GetAllOwnedAssets(model->getWallet(), names);
+    for (auto item : names) {
+        std::string name = QString::fromStdString(item).split("!").first().toStdString();
+        if (name.size() != 30)
```

Comment:
What is this check for?  I think we could have a 30-char asset name (plus !)..

Reply:
Yeah. So, I am splitting the ! off of these  assets names. Then after that if they are < 30 characters. They won't show up because the asset name is already at the limit